### PR TITLE
feat: medication request by id rpc call

### PR DESCRIPTION
**Pull request by BatrakM**

### CODE REVIEW

File: apps/ops/lib/ops/rpc.ex [hunk 1]
```diff
@@ -324,4 +357,183 @@ defmodule OPS.Rpc do
       err -> err
     end
   end
+
+  @doc """
+  Get medication request by id
+
+  ## Examples
+      iex> OPS.Rpc.medication_request_by_id("0469f379-ff2e-4a69-81e8-2cbfadf88d6b")
+      %Core.MedicationRequests.MedicationRequest{
```
Comment (progsmile) [inline apps/ops/lib/ops/rpc.ex]: It's getting inconsistent when microservice can return both **structs** or **rendered maps**
Should render structs, yes @AlexKovalevych?

File: apps/ops/lib/ops/rpc.ex [hunk 2]
```diff
@@ -324,4 +357,183 @@ defmodule OPS.Rpc do
...
+        updated_at: ~N[2019-02-05 09:35:00.027131],
+        updated_by: "4261eacf-8008-4e62-899f-de1e2f7065f0",
+        verification_code: "7291"
+      }
+  """
+  @spec medication_request_by_id(binary) :: medication_request | nil
+  def medication_request_by_id(medication_request_id) do
+    MedicationRequest
+    |> where([mr], mr.id == ^medication_request_id)
+    |> @read_repo.one()
```
Comment (AlexKovalevych) [inline apps/ops/lib/ops/rpc.ex]: you should render this to regular map, no need to return a struct

File: apps/ops_scheduler/test/jobs/declarations_approve_test.exs [hunk 3]
```diff
@@ -18,7 +18,7 @@ defmodule OpsScheduler.Jobs.DeclarationsApproveTest do
 
     declaration_ids =
       Enum.map(1..10, fn _ ->
-        inserted_at = NaiveDateTime.add(NaiveDateTime.utc_now(), -86_400 * 10, :seconds)
+        inserted_at = NaiveDateTime.add(NaiveDateTime.utc_now(), -86_400 * 10, :second)
```
Comment (progsmile) [inline apps/ops_scheduler/test/jobs/declarations_approve_test.exs]: And this one has `DateTime` type

File: apps/core/test/support/factory.ex [hunk 4]
```diff
@@ -10,8 +10,8 @@ defmodule Core.Factory do
   alias Ecto.UUID
 
   def declaration_factory do
-    start_date = NaiveDateTime.utc_now() |> NaiveDateTime.add(days_to_seconds(-10), :seconds)
-    end_date = NaiveDateTime.add(start_date, days_to_seconds(1), :seconds)
+    start_date = NaiveDateTime.utc_now() |> NaiveDateTime.add(days_to_seconds(-10), :second)
+    end_date = NaiveDateTime.add(start_date, days_to_seconds(1), :second)
```
[the remark this answers is not in the document]
Comment (BatrakM) [reply apps/core/test/support/factory.ex]: done

File: apps/ops/lib/ops/rpc.ex (hunk 1 above)
Comment (BatrakM) [reply apps/ops/lib/ops/rpc.ex]: done

File: apps/ops/lib/ops/rpc.ex (hunk 2 above)
Comment (BatrakM) [reply apps/ops/lib/ops/rpc.ex]: done

**Pull request merged**